src/pages/AMM/Pools/components/PoolModal/PoolModal.tsx

```diff
@@ -47,11 +47,13 @@ const PoolModal = ({ pool, onClose, ...props }: PoolModalProps): JSX.Element | n
               <DepositForm slippageModalRef={ref} pool={pool} onDeposit={handleAction} />
             </StyledWrapper>
           </TabsItem>
-          <TabsItem title={t('withdraw')}>
-            <StyledWrapper>
-              <WithdrawForm slippageModalRef={ref} pool={pool} onWithdraw={handleAction} />
-            </StyledWrapper>
-          </TabsItem>
+          {!pool.isEmpty && (
```

Comment:
![image](https://user-images.githubusercontent.com/43269067/235892635-ecad14e3-0603-4dab-aa4f-1a89a7f90cca.png)

This is what I suggest instead of removing it. For this to be implemented merging the following PR is required: https://github.com/interlay/interbtc-ui/pull/1178

Reply:
This was approved by Jay